pyproject.toml

```diff
@@ -1,16 +1,50 @@
 [build-system]
-requires = [
-  "setuptools",
-]
+requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "autopxd2"
+version = "2.4.1"
+description = "Automatically generate Cython pxd files from C headers"
+readme = "README.md"
+license = { file = "LICENSE" }
+authors = [
+  { name = "elijahr", email = "elijahr+autopxd2@gmail.com" },
+  { name = "touilleMan", email = "emmanuel.leblond@gmail.com" },
+  { name = "Gabriel de Marmiesse", email = "gabrieldemarmiesse@gmail.com" },
+  { name = "Thiago de Arruda", email = "tpadilha84@gmail.com" },
+]
+urls = { "Homepage" = "https://github.com/elijahr/python-autopxd2" }
+keywords = ["autopxd", "autopxd2", "cython", "code-generators"]
+classifiers = [
+  "Topic :: Software Development :: Compilers",
+  "Topic :: Software Development :: Code Generators",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Cython",
+  "Programming Language :: C",
+  "Programming Language :: C++",
+]
+dependencies = ["Click", "pycparser", "setuptools", "wheel"]
```

Comment:
Do we still depend on setuptools ?

I would say this is only the case since we use `pkg_resources`, however [it is considered is deprecated](https://setuptools.pypa.io/en/latest/pkg_resources.html) and `importlib.resources`/`importlib.metadata` is advised to be used instead.

(This PR is big enough, if you agree with this change we may want to open an issue for now, and solve it later in a separated PR)